### PR TITLE
chore: updated lingvist with new details for account deletion

### PR DESCRIPTION
**Pull request by ohheyrj**
Updated lingvist to easy as you can now delete your account from within the website and app. Turkish translation supplied by Google Translate

### CODE REVIEW

File: _data/sites.json [hunk 1]
```diff
@@ -6460,11 +6460,11 @@
...
         "name": "Lingvist",
-        "url": "https://lingvist.helpscoutdocs.com/article/173-i-dont-want-to-pay-delete-my-account",
+        "url": "https://lingvist.com/help/how-to-delete-my-account-and-download-my-data/",
         "email": "hello@lingvist.io",
-        "difficulty": "hard",
-        "notes": "You must contact Lingvist to request account deletion.",
-        "notes_tr": "Hesap silme talebinde bulunmak için Lingvist ile iletişime geçmelisiniz.",
+        "difficulty": "easy",
+        "notes": "You can delete your account via the web or mobile app.",
+        "notes_tr": "Hesabınızı web veya mobil uygulama üzerinden silebilirsiniz.",
```
Comment (tupaschoal) [inline _data/sites.json:6467]:
```suggestion
```
We don't rely on machine translated descriptions, so whenever a note is updated, we delete the translations unless we know the other languages ourselves.

**Review by tupaschoal — COMMENTED** [review]
Looks good, just one note about translations

File: _data/sites.json (hunk 1 above)
Comment (ohheyrj) [reply _data/sites.json]: @tupaschoal I've removed the translation as per above :)

**Review by tupaschoal — APPROVED** [review]

**Comment by tupaschoal** [comment]
Thanks!

**Pull request merged**